ethereum/contracts/BeefyLightClient.sol

```diff
@@ -63,6 +58,19 @@ contract BeefyLightClient {
         uint32 validatorSetId;
     }
 
+    /**
```

Comment:
Please add a comment how this struct is used

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/dcc5b1468ca5e3618b1c07254dbb501b44aae014